docs/UserGuide.md

```diff
@@ -649,21 +648,24 @@ SUCCESS: Medicine has been deleted
 
 ## Search
 
-To search for a medication in the library, you can type 'search', followed by the field you want to search for and the keyword 
-`keyword`.
+MediTracker has a built-in medication library where you can search for common medications. This library contains 
+information on the medication name, the illness that the medication is used for, and the side effects of the medication.
+
```

Comment:
Perhaps you can add a Note admonition box here to somehow alert confused users that this search command is searching in another file, and not in the current medication list in 'list -t all'